functests/utils/utils.go

```diff
@@ -43,3 +44,32 @@ func ExecAndLogCommand(name string, arg ...string) ([]byte, error) {
 	}
 	return out, err
 }
+
+func ExecAndLogCommandWithStderr(name string, arg ...string) ([]byte, []byte, error) {
```

Comment:
Can you update `ExecAndLogCommand` to use your new command?

Reply:
sure, on it